docs/custom_operation.rst

```diff
@@ -146,3 +146,30 @@ to use the `NGC PyTorch container
 corresponding to the Triton container. For example, if you are using
 the 20.03 version of Triton, use the 20.03 version of the PyTorch
 container.
+
+ONNX
+-------
+
+ONNXRuntime allows users to `add custom operations
+<https://github.com/microsoft/onnxruntime/blob/master/docs/AddingCustomOp.md>`_
+which can then be used in ONNX models. To register your custom operations
+library you need to include it in the model configuration as an additional
+field. For example, if you follow `this example
+<https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/test/shared_lib/test_inference.cc>`_
+in the `microsoft/onnxruntime<https://github.com/microsoft/onnxruntime>`_
+repository and your ONNXRuntime custom operations are compiled into libonnxcustom.so,
+adding the following to the model configuraion of your model makes those 
+operations available to that specific ONNX model::
+
+  $ model_operations { op_library_filename: "/path/to/libonnxcustom.so" }
+
+A limitation of this approach is that the custom operations must be
```

Comment:
The first part of this paragraph is not true given that you can have the onnx shared library inside the model repo. Is the second part true? Does ONNXRT have a shared namespace for all custom operations? Even though you are loading them per-model? That would be strange.

Reply:
You are right. This paragraph is not relevant for ONNXRT